src/Gui/BitmapFactory.cpp

```diff
@@ -37,9 +37,6 @@
 # include <QStyleOption>
 # include <sstream>
 #endif
-#if defined (FC_OS_WIN32) && QT_VERSION < 0x050000
-#define QTWEBKIT
-#endif
 
 #ifdef QTWEBKIT

```

Comment:
The whole block
```c++
#ifdef QTWEBKIT
...
#endif QTWEBKIT
```
can be removed. This was a workaround with Qt4 because the QtSvg module was very limited and often failed to create a valid icon. So, we used the Webkit engine to create the icons.
With Qt5 QtSvg is much more powerful and creates valid icons for actually all plain SVG files